recipes/default.rb

```diff
@@ -17,3 +17,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+case node['platform_family']
```

Comment:
This probably isn't the desired behavior and we should instead rely on users to add the desired individual recipes.